.github/workflows/markdown-link-check.yaml

```diff
@@ -0,0 +1,32 @@
+---
+"on":
+  push:
+    branches:
+      - trunk
+    paths:
+      - .github/markdown-link-check.json
+      - .github/workflows/markdown-link-check.yaml
+      - "**/*.md"
+  pull_request:
+    branches:
+      - trunk
+    paths:
+      - .github/markdown-link-check.json
+      - .github/workflows/markdown-link-check.yaml
+      - "**/*.md"
+  schedule:
+    - cron: "0 0 * * TUE"
+name: Markdown Links Check
+jobs:
+  check-links:
+    name: Check links
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: gaurav-nelson/github-action-markdown-link-check@v1
```

Comment:
this will probably fail the first time up due to markdown in vendored sources.

Maybe we can do this in a preceding step?

```sh
shopt -s globstar
rm -rf **/vendor/*/*
```

I think this will keep all of the `vendor/*.md` files in tact.

Reply:
It looks like that'll work:

```console
$ /usr/local/bin/bash --version
GNU bash, version 5.1.12(1)-release (x86_64-apple-darwin20.6.0)
Copyright (C) 2020 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
$ /usr/local/bin/bash -c 'shopt -s globstar ; rm -rf **/vendor/*/' # use bash 5
$ ls artichoke-backend/vendor/
README.md
```